scripts/publish-assets.sh

```diff
@@ -10,4 +10,4 @@ fi
 
 export COMMIT_HASH=`cat COMMIT_HASH.txt`
 
-yarn assets && yarn bucket-assets -b $S3_BUCKET
+yarn assets && ./node_modules/.bin/bucket-assets -b $S3_BUCKET
```

Comment:
Why is this change necessary?

Reply:
I was worried that the name change might affect this command, but might not be necessary -- will remove and see if it works as expected on staging deploy.

Reply:
Nope, the `@foo` is a npm namespace scope: https://docs.npmjs.com/about-scopes for grouping packages. 
 What matters is the `node_modules/.bin/file-name` reference as those are what's made available to npm or yarn via `yarn x` commands.